### PR TITLE
Fixes #85 appveyor issue

### LINKED ISSUE #85: AppVeyor issue
@msevestre Do you have any experience with error like this in `AppVeyor` builds? 
I've never experienced something like this before, so not sure how to fix it.

https://ci.appveyor.com/project/IndrajeetPatil/esqlabsr/builds/40950527/job/4hm67gc36c7imgfk#L963

image

The package version is completely fine, so not sure why it thinks it's malformed:
https://github.com/esqLABS/esqlabsR/blob/42a85204ce4e0b8c479d69579d84a7633c348b1f/DESCRIPTION#L4

### PR DESCRIPTION
**Pull request by msevestre**

### CODE REVIEW

File: rakefile.rb [hunk 1]
```diff
@@ -1,197 +1,14 @@
...
 require_relative 'scripts/utils'
 require_relative 'scripts/R'
-require_relative 'scripts/colorize'
-
-OpenSSL::SSL::VERIFY_PEER = OpenSSL::SSL::VERIFY_NONE
-
-APPVEYOR_ACCOUNT_NAME = 'open-systems-pharmacology-ci'
 
 task :prepare_for_build, [:build_version] do |t, args|
   update_package_version(args.build_version, description_file)
```
Comment (msevestre) [inline rakefile.rb:5]: That should be the only thing required. it will update the package version 2.0.0 to be 2.0.BUILD NUMBER

**Comment by IndrajeetPatil** [comment]
Seems to be working now!!! 🎉

File: appveyor.yml [hunk 2]
```diff
@@ -15,32 +12,28 @@ install:
...
-  - nuget sources add -name funcparser -source https://ci.appveyor.com/nuget/ospsuite-funcparser
-  - nuget sources add -name simmodel -source https://ci.appveyor.com/nuget/ospsuite-simmodel
-  - nuget sources add -name cvodes -source https://ci.appveyor.com/nuget/ospsuite-simmodel-solver-cvodes
-  - nuget sources add -name core -source https://ci.appveyor.com/nuget/ospsuite-core
-  - nuget sources add -name centos -source https://ci.appveyor.com/nuget/ospsuite-centos
-  - nuget install packages.config -OutputDirectory packages # cannot use automatic artifact publish because it's not recognized in a C++/CLI project
   - rake "prepare_for_build[%APPVEYOR_BUILD_VERSION%]"
 
 build_script:
   - Rscript -e "install.packages('./tools/rClr_0.9.1.zip', repos = NULL, type = 'binary')"
```
Comment (PavelBal) [inline appveyor.yml:32]: Can we reference the online version?:

`Rscript -e "install.packages('.https://github.com/Open-Systems-Pharmacology/rClr/releases/download/v0.9.1/rClr_0.9.1.zip', repos = NULL, type = 'binary')"`

It works locally in RStudio. This way we do not have to pack rClr in the package.

File: rakefile.rb [hunk 3]
```diff
@@ -12,73 +11,10 @@
...
-def copy_so(file, linux_distro, target_dir)
-  native_folder = '/bin/native/x64/Release/'
-  copy_dependencies packages_dir, target_dir do
-    copy_files "#{file}.#{linux_distro}*/**/#{native_folder}", 'so'
-    copy_files "#{file}.#{linux_distro}*/**/netstandard*", 'dll'
-  end
-end
-
 def install_pksim(branch)
   file_name ='setup.zip'
```
Comment (PavelBal) [inline rakefile.rb:19]: I think PK-Sim is required for some tests that use creation of individuals. Maybe we should consider at some point to completely remove the dependency by separating the "create..." algorithms in separate submodule?

**Review by PavelBal — CHANGES_REQUESTED** [review]
Just one comment regarding using rClr from github repo.

**Comment by msevestre** [comment]
Lets' try

**Comment by msevestre** [comment]
that works with rclr remote... damm... Going to do this for ospsuite-r too!

**Comment by IndrajeetPatil** [comment]
Woot!! You sir are a genius! Thanks for getting it to work. 😅

**Comment by msevestre** [comment]
ok I have added the badge for CI also (coming from Appveyor). I think this is ready to merge

**Pull request merged**